src/app/app.component.scss

```diff
@@ -0,0 +1,7 @@
+
+.box{
+  margin: auto;
+  width: 70vw;
+  height: 70vh;
+  border: 1px #f4f4f4 solid;
```

Comment:
for colors, it would be best to create a file in assets/styles named colors.scss
and use css variables there - this way you can reuse them in whole app